### PR TITLE
Feature-gate #[prelude_import].

### LINKED ISSUE #26690: prelude_import should be unstable?
cc @chris-morgan, @eddyb 

If this is a bug, we should fix it asap.

### PR DESCRIPTION
**Pull request by eddyb**
Closes #26690.

### CODE REVIEW

**Comment by alexcrichton** [comment]
cc @huonw, you seem pretty familiar with the `allow_internal_unstable` stuff.

This may also be good to run through Crater first just to double check, but I'd also be fine just going ahead and merging this and covering it in one of the regular "regressions against beta" reports.

Also, nominating for a beta backport.

File: src/libsyntax/std_inject.rs [hunk 1]
```diff
@@ -10,16 +10,35 @@
...
+        call_site: DUMMY_SP,
+        callee: NameAndSpan {
+            name: "std_inject".to_string(),
+            format: MacroAttribute,
+            span: None,
+            allow_internal_unstable: true,
+        }
+    };
+    let expn_id = sess.codemap().record_expansion(info);
+    let mut sp = sp;
```
Comment (huonw) [inline src/libsyntax/std_inject.rs]: Could be `Span { expn_id: expn_id, .. sp }`?

**Comment by huonw** [comment]
r=me with an passing crater run.

**Comment by brson** [comment]
I will start now.

**Comment by brson** [comment]
[Crater says all good](https://gist.github.com/brson/ed71843cd3e84fd97759).

**Comment by brson** [comment]
Not accepted for backport. People aren't using this and we can break it at will. Bugfix.

**Pull request merged**